test-data/unit/semanal-errors.test

```diff
@@ -1330,3 +1330,13 @@ a = ('spam', 'spam', 'eggs', 'spam')  # type: Tuple[str]
 [out]
 main:3: error: Name 'a' already defined
 main:4: error: Name 'a' already defined
+
+[case testUnknownContextDuplicateDef]
+from m import A
+class A:
+    pass
+[file m.py]
+class A:
+    pass
+[out]
+main:2: error: Name 'A' already defined (line unavailable; possibly an import)
```

Comment:
Yes, but that will cause a completely different error message (types are not compatible). It won't get to the point of complaining about duplicate definition.